src/server/service/achievements.ts

```diff
@@ -42,18 +32,24 @@ const busterStreakAchievement: Achievement = {
   name: "Buster Streak",
   description: "Bust three bots in a row",
   calculate: ({ player, playerHistory, otherPlayers, botsBusted }) => {
-    if (
+    const isNotEligibleForNewAchievement =
       !playerHistory ||
       playerHistory.length < 2 ||
```

Comment:
why the length check?

Reply:
Because we do not have loop over the data if the player not even had played 3 matches to begin with.